src/main/cliapp/src/containers/affectedGenomicModelPage/AffectedGenomicModelTable.js

```diff
@@ -46,30 +47,30 @@ export const AffectedGenomicModelTable = () => {
 			header: "Curie",
 			sortable: isEnabled,
 			filter: true,
-			filterElement: {type: "input", filterName: "curieFilter", fields: ["curie"]},
+			filterElement: {type: "input", filterName: "curieFilter", fields: FILTER_FIELDS.curieFilter},
```

Comment:
@adamgibs so... you should be able to lookup the filter via name and not have to pass `fields` in all these methods.

Reply:
@oblodgett Sure, refactoring the filter elements and centralizing code for autocomplete makes sense. However, this was just for the local storage filter fields validation bug. I just switched the fields so that the validation logic and the filtering logic were pointing to the same map.